LayoutTests/imported/w3c/web-platform-tests/css/css-anchor-position/anchor-query-custom-property-registration-expected.txt

```diff
@@ -1,5 +1,10 @@
 
-PASS anchor() cannot be used as <length> initial value
+FAIL anchor() cannot be used as <length> initial value assert_throws_dom: function "() => CSS.registerProperty({
+    name: '--x',
+    syntax: '<length>',
+    inherits: false,
+    initialValue: 'anchor(--foo top)',
+  })" did not throw
```

Comment:
Would it be easy to fix this?

Reply:
It should be fairly trivial but it is better handled separately.